torch/_dynamo/variables/builtin.py

```diff
@@ -1557,7 +1596,12 @@ def call_cast(self, _, *args, **kwargs):
         if len(args) == 2:
             return args[1]
 
-        unimplemented(f"unsupported args to builtin cast(): {args} {kwargs}")
+        unimplemented_v2(
+            gb_type="bad args to builtin cast()",
+            context=f"got args {args} {kwargs}",
+            explanation="Dynamo expects exactly 2 args to builtin cast()",
+            hints=[],
```

Comment:
I think this is one of those odd cases where things would fail in eager as well, maybe just add a hint to suggest that?